reposerver/repository/repository.proto

```diff
@@ -26,6 +26,7 @@ message ManifestRequest {
     repeated string apiVersions = 15;
     // Request to verify the signature when generating the manifests (only for Git repositories)
     bool verifySignature = 16;
+    repeated github.com.argoproj.argo_cd.v2.pkg.apis.application.v1alpha1.RepoCreds repoCreds = 17;
```

Comment:
Both `repos = 11` and `repoCreds = 17` fields have only Helm repositories. It is different from Git because Helm has dependencies, so we have to pass all Helm creds to repo server.

The `repos = 11` was named incorrectly - it should be `helmRepos` and `repoCreds` should be `helmRepoCreds`. Can you rename fields accordingly? 

Please try to keep API backward compatible. E.g. keep `repos = 11` but prefer value from new `helmRepos`

Reply:
I will leave the refactor of repo to helmRepos in a separate PR to make this PR simple.

Reply:
Sounds good 👍